### DISCUSSION TITLE
ROCm 6.2.2 Release

### DISCUSSION THREAD

**Discussion by samjwu** [ROCm/ROCm#3829] [Announcements]
# ROCm 6.2.2 release notes

These release notes provide a summary of notable changes since the previous ROCm release.

```{note}
As ROCm 6.2.2 was released shortly after 6.2.1, the changes between these versions
are minimal. For a comprehensive overview of recent updates,
refer to the ROCm 6.2.1 release notes.
```

The [Compatibility matrix](https://rocm.docs.amd.com/en/docs-6.2.2/compatibility/compatibility-matrix.html)
provides the full list of supported hardware, operating systems, ecosystems, third-party components, and ROCm components
for each ROCm release.

Release notes for previous ROCm releases are available in earlier versions of the documentation.
See the [ROCm documentation release history](https://rocm.docs.amd.com/en/latest/release/versions.html).

## Release highlights

The following is a significant fix introduced in ROCm 6.2.2.

### Fixed Instinct MI300X error recovery failure

Improved the reliability of AMD Instinct MI300X accelerators in scenarios involving
uncorrectable errors. Previously, error recovery did not occur as expected,
potentially leaving the system in an undefined state. This fix ensures that error
recovery functions as expected, maintaining system stability.

See the original issue noted in the ROCm 6.2.1 release notes.

<hr /><em>This discussion was created from the release <a href='https://github.com/ROCm/ROCm/releases/tag/rocm-6.2.2'>ROCm 6.2.2 Release</a>.</em>

**Comment by ZJLi2013** [comment]
can we provide a rocm:6.2.x docker image public link in future ?
